index.js

```diff
@@ -317,10 +317,12 @@ Router.prototype.off = function off (method, path) {
   })
 }
 
-Router.prototype.lookup = function lookup (req, res) {
+Router.prototype.lookup = function lookup (req, res, ctx) {
   var handle = this.find(req.method, sanitizeUrl(req.url), req.headers['accept-version'])
-  if (handle === null) return this._defaultRoute(req, res)
-  return handle.handler(req, res, handle.params, handle.store)
+  if (handle === null) return this._defaultRoute(req, res, ctx)
+  return ctx == null
```

Comment:
I think here we can use a stricter check :)
```suggestion
  return ctx === undefined
```